libpod/runtime_ctr.go

```diff
@@ -96,6 +96,13 @@ func (r *Runtime) NewContainer(rSpec *spec.Spec, options ...CtrCreateOption) (c
 			return nil, err
 		}
 	}
+
+	if ctr.config.ImageVolumes {
```

Comment:
This is not a boolean but  alist, Shouldn't you be checking size?

Reply:
No, this is a boolen. This is the one in the container config.